ekf:
  output:
    topic_name: /estimate/ekf
  init_values:
    x0: [0, 0, 0, 0, 0, 0, 0]
    P0: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
  dt: 0.01

estimators:
  dnn_cv:
    output:
      topic_name: /estimate/drone_pose/dnnCV
    sigmas: [0.1, 0.1, 0.1, 0.1] # std. dev. in x,y,z,psi